### DISCUSSION TITLE
Single sided border

### DISCUSSION THREAD

**Discussion by Fabi** [dotnet/maui#6433] [Q&A]
As far as I can tell there is not simple way to assign a single sided border to a control. And the Border control only supports settings StrokeThickness for all 4 sides. Is there any possibility I miss here? If not it would be nice to have that implemented.

**Comment by saint4eva** [comment]
new Thickness (5,0,0,0)

**Reply by Fabi** [reply]
> new Thickness (5,0,0,0)

Won't work. It's a double in MAUI

**Reply by saint4eva** [reply]
5.0, 0.0, 0.0, 0.0

**Reply by Fabi** [reply]
> 5.0, 0.0, 0.0, 0.0

NO.

it's not a Thickness. It's a damn double...

**Reply by saint4eva** [reply]
It seems you do not know what you actually want.

**Reply by Fabi** [reply]
> It seems you do not know what you actually want.

You don't seem to know how borders work in maui

**Reply by KarlSearl** [reply]
instead of using a border, try using a box view, we a trick we used in xamarin forms.
